compiam/data.py

```diff
@@ -36,7 +36,7 @@
         "class_name": "FTANetCarnatic",
         "kwargs": {
             "model_path": os.path.join(
-                WORKDIR, "models", "melody", "ftanet", "carnatic", "OA"
+                WORKDIR, "models", "melody", "ftanet", "carnatic", "carnatic"
```

Comment:
carnatic/carnatic?

Reply:
First carnatic is the folder, while second carnatic is the name of the model. We could remove one level, but if at some point we would like to add more FTANet weights (e.g. hindustani), we would need to have all model files in the same folder (`ftanet/`), which is a little bit messy maybe.